tools/hangar/src/invalid.test.ts

```diff
@@ -19,7 +19,7 @@ describe.each(invalidWingFiles)("%s", (wingFile) => {
 
       const out = await runWingCommand(testDir, relativeWingFile, args, false);
 
-      const stderr = out.all ?? out.stderr;
+      const stderr = out.stderr;
```

Comment:
Why not "all"?

Reply:
It wasn't really behaving as expected. It would often be empty even though there was definitely something in stderr at least. Maybe it didn't like when stdout was empty.